main.go

```diff
@@ -601,6 +603,9 @@ func main() {
 		}
 	case "help":
 		usage()
+	case "version":
+		fmt.Fprintln(os.Stderr, displayversion())
```

Comment:
The name `displayversion` suggests to me that this function actually prints the version.